cmd/minikube/cmd/start.go

```diff
@@ -390,6 +390,14 @@ func validateConfig() {
 		glog.Errorf("Error autoSetOptions : %v", err)
 	}
 
+	memorySizeMB := pkgutil.CalculateSizeInMB(viper.GetString(memory))
+	if memorySizeMB < pkgutil.CalculateSizeInMB(constants.MinimumMemorySize) {
+		exit.Usage("Requested memory allocation (%dMB) is less than the minimum allowed of %dMB", memorySizeMB, pkgutil.CalculateSizeInMB(constants.MinimumMemorySize))
+	}
+	if memorySizeMB < pkgutil.CalculateSizeInMB(constants.DefaultMemorySize) {
+		console.OutStyle(console.Notice, "Requested memory allocation (%dMB) is less than the default memory allocation of (%dMB). Beware that Minikube might not work correctly or crash unexpectedly.", memorySizeMB, pkgutil.CalculateSizeInMB(constants.DefaultMemorySize))
```

Comment:
pleaese change Minikube to minikube (we are mini no need to capitalize :) )

Reply:
Done.